src/core.js

```diff
@@ -380,6 +390,10 @@ $.extend( $.validator, {
 				rules[ key ] = $.validator.normalizeRule( value );
 			} );
 
+			// Delete `onkeyup` if `input` event is supported
+			// Otherwise, remove `oninput` handler
+			delete this.settings[ inputSupported ? "onkeyup" : "oninput" ];
+
```

Comment:
As discussed above, one of the 2 handlers (`onkeyup` and `oninput` should be removed based on `input` event support).